src/pages/index.tsx

```diff
@@ -30,7 +30,8 @@ function HomepageHeader() {
             </div>
           </div>
           <div className="col">
-            <img src="/img/temp_screenshot.png" alt="Regression Games" style={{borderRadius: "12px"}}/>
+            <img src={require('@site/static/img/demo.gif').default} alt="Regression Games" style={{borderRadius: "12px"}}/>
+            <div style={{color: "gray", paddingRight: "24px", fontStyle: "italic"}}>Demo of bots validating abilities in Boss Room, the timeline feature, and the automated Agent Builder.</div>
```

Comment:
I still think we need to link to BossRoom (https://unity.com/demos/small-scale-coop-sample) and give Unity team more credit/reference for this to clarify this is not our own work.

Reply:
I think after crediting them, we could also add a pointed to our RGBossRoom project as a demo of a more full integration.  But maybe add that later after this first customer interaction

Reply:
Fixed! And yeah I agree